modules/oe_content_entity/tests/src/Kernel/CorporateEntityAccessControlHandlerTest.php

```diff
@@ -30,6 +30,20 @@ class CorporateEntityAccessControlHandlerTest extends EntityKernelTestBase {
    */
   protected $accessControlHandler;
 
+  /**
+   * The corporate entity type access control handler.
+   *
+   * @var \Drupal\oe_content_entity\CorporateEntityTypeAccessControlHandler
+   */
+  protected $bundleAccessControlHandler;
+
+  /**
+   * A corporate entity to be used in tests.
+   *
+   * @var \Drupal\oe_content_entity\Entity\CorporateEntityTypeInterface
+   */
+  protected $entity;
```

Comment:
I don't see why we need this global variable at all. You create an entity during setup, that is being used only in one test case so far, for other test cases it's not being used but still it is being created.